snake_ai.py

```diff
@@ -34,13 +35,17 @@ def _best_move_helper(self, snake_id, board, depth=DEPTH):
         given board, according to the heuristic, calculated to the given depth.
         Assume other snakes make the best immediate move.
         """
-        move_evals = {
-            move: self._get_move_eval(
-                snake_id,
-                self.game.simulate_moves(board, {snake_id: move}),
-                depth)
-            for move in Move}
-        return max(move_evals.items(), key=lambda x: x[1])
+        MoveEvalPair = namedtuple('MoveEvalPair', "move, evaluation")
+        move_evals = \
```

Comment:
Stop using line continuation like this. 

If you use brackets/parentheses it is automatically a line continuation.

It also looks plain ugly.

instead.

` = [
MoveEval...

]`